crates/bevy_image/src/dynamic_texture_atlas_builder.rs

```diff
@@ -18,6 +18,9 @@ pub enum DynamicTextureAtlasBuilderError {
     /// Attempted to add an uninitialized texture to an atlas
     #[error("cannot add uninitialized texture to atlas")]
     UninitializedSourceTexture,
+    /// A texture access error occurred
+    #[error("texture access error: {0}")]
```

Comment:
Capitalization is inconsistent across these error types. I don't know if we've defined a style 🤔